packages/aws-rfdk/lib/core/test/cloudwatch-agent.test.ts

```diff
@@ -60,32 +58,10 @@ describe('CloudWatchAgent', () => {
     });
 
     // THEN
-    expectCDK(stack).to(haveResource('AWS::SSM::Parameter', {
+    Template.fromStack(stack).hasResourceProperties('AWS::SSM::Parameter', {
       Type: 'String',
       Value: cloudWatchConfig,
-    }));
-  });
-
-  test('creates an asset', () => {
```

Comment:
Note: This test is not possible with the under-the-hood changes introduced in CDKv2. Fortunately, this is tested, in a way, by the tests that look at UserData; the CDK now generates the file sha256 before synthesis, so file hashes are directly embedded into UserData in both the rendered stack and when directly rendering the UserData string from the contruct.